src/components/link/ProjectChildPageLink.js

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+
+export const getLinkData = (childPage, projectSlug) => ({
+  urlObject: {
+    pathname: `/projects/[projectSlug]/[projectSlug]-[childPage]`,
```

Comment:
same thing with the pathName, it should be the same as it's in the app.render in server.js